app/controllers/games_controller.rb

```diff
@@ -3,6 +3,12 @@ def new
     @game = Game.new
   end
 
+  def check
+    @game = current_game
+    @game.check?
+    flash[:notice] = 'You are in Check'
+  end
+
```

Comment:
Check to see if status of current game is in check? 
and then render flash notice